examples/multi-az/main.tf

```
@@ -0,0 +1,67 @@
+provider "aws" {
+  region = local.region
+}
+
+locals {
+  name   = "ex-${replace(basename(path.cwd), "_", "-")}"
+  region = "eu-west-1"
+
+  tags = {
+    Example    = local.name
+    GithubRepo = "terraform-aws-rds-aurora"
+    GithubOrg  = "terraform-aws-modules"
+  }
+}
+
+################################################################################
+# RDS Aurora Module
+################################################################################
+
+module "aurora" {
+  source = "../../"
+
+  name           = local.name
+  engine         = "postgres" # This uses RDS, not Aurora
+  engine_version = "13.7"
+
+  vpc_id  = module.vpc.vpc_id
+  subnets = module.vpc.private_subnets
+
+  create_db_cluster_parameter_group = true
+  db_cluster_parameter_group_family = "postgres13"
+  enabled_cloudwatch_logs_exports   = ["postgresql"]
+
+  # Multi-AZ
+  availability_zones        = module.vpc.azs
+  allocated_storage         = 256
+  db_cluster_instance_class = "db.r6gd.xlarge"
```

```suggestion
  db_cluster_instance_class = "db.r6gd.large"
```
Looks a bit cheaper, and still sufficient for this example according to https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/Concepts.DBInstanceClass.html